src/globus_sdk/services/flows/client.py

```diff
@@ -820,7 +820,7 @@ class SpecificFlowClient(client.BaseClient):
     Client for interacting with a specific Globus Flow through the Flows API.
 
     Unlike other client types, this must be provided with a specific flow id. All other
-        arguments are the same as those for `~globus_sdk.BaseClient`.
+        arguments are the same as those for :class:`~globus_sdk.BaseClient`.
```

Comment:
Great catch! I have a sneaking suspicion that we'll find this same mistake elsewhere in the docs too.